api/v1/model_handlers.go

```diff
@@ -318,18 +318,27 @@ func (a *Api) containerPaths(request *restful.Request, response *restful.Respons
 // allNodes returns a list of all the available node names in the cluster.
 func (a *Api) allNodes(request *restful.Request, response *restful.Response) {
 	cluster := a.manager.GetCluster()
+	if cluster == nil {
+		response.WriteErrorString(400, "The model is not activated")
```

Comment:
In fact, for the sole purpose of deduplicating the error string you could just have the error string defined as a const:

```go
const errModelNotActivated = errors.New("the model is not activated")
[...]
response.WriteError(400, errModelNotActivated)
```

Which is much simpler than adding the new method.